scripts/controlnet.py

```diff
@@ -1370,30 +1367,18 @@ def on_ui_settings():
         1, "Multi ControlNet: Max models amount (requires restart)", gr.Slider, {"minimum": 1, "maximum": 10, "step": 1}, section=section))
     shared.opts.add_option("control_net_model_cache_size", shared.OptionInfo(
         1, "Model cache size (requires restart)", gr.Slider, {"minimum": 1, "maximum": 5, "step": 1}, section=section))
-    shared.opts.add_option("control_net_control_transfer", shared.OptionInfo(
-        False, "Apply transfer control when loading models", gr.Checkbox, {"interactive": True}, section=section))
     shared.opts.add_option("control_net_no_detectmap", shared.OptionInfo(
         False, "Do not append detectmap to output", gr.Checkbox, {"interactive": True}, section=section))
-    shared.opts.add_option("control_net_high_res_only_mid", shared.OptionInfo(
-        False, "Use only-mid-control on high-res. fix (second pass)", gr.Checkbox, {"interactive": True}, section=section))
     shared.opts.add_option("control_net_detectmap_autosaving", shared.OptionInfo(
         False, "Allow detectmap auto saving", gr.Checkbox, {"interactive": True}, section=section))
     shared.opts.add_option("control_net_allow_script_control", shared.OptionInfo(
         False, "Allow other script to control this extension", gr.Checkbox, {"interactive": True}, section=section))
-    shared.opts.add_option("control_net_skip_img2img_processing", shared.OptionInfo(
```

Comment:
Is there a good reason this option was removed? I had been actively using it and relying on it in my workflows. Removed w/o explanation and AFAIK no other way to reproduce this functionality. This feature was very useful for running images directly through controlNET without doing the initial img2img processing.

Reply:
You can use controlnet batch instead now, it works both in img2img and txt2img tabs. It will soon be moved into its own accordion.